source/change-streams/change-streams.rst

```diff
@@ -278,6 +278,19 @@ Driver API
      * @note this is an option of the `$changeStream` pipeline stage.
      */
     startAtOperationTime: Optional<Timestamp>;
+
+    /**
+     * Similar to 'resumeAfter', this option takes a resume token and starts a
+     * new change stream returning the first notification after the token.
+     * This will allow users to watch collections that have been dropped and recreated
+     * or newly renamed collections without missing any notifications.
+     *
+     * The server will report an error if startAfter and resumeAfter are both specified.
```

Comment:
nit: wrap `startAfter` and `resumeAfter`

Reply:
Do you mean wrap in backticks? If so, perhaps that should be done for resumeAfter a few lines above as well. It currently uses single quotes.

Reply:
yeah let's wrap them all in backticks to indicate that they are known property names